.buildkite/configs/cleanup.aws.yml

```diff
@@ -5,8 +5,8 @@ accounts:
   - name: "${ACCOUNT_PROJECT}"
     driver: "aws"
     options:
-      key: '${ACCOUNT_KEY}'
-      secret: '${ACCOUNT_SECRET}'
+      key: '${AWS_ACCESS_KEY_ID}'
```

Comment:
To help with using the existing env variables created by https://github.com/elastic/oblt-aws-auth-buildkite-plugin